blueprint/core/blueprint_EcmascriptEngine.cpp

```diff
@@ -95,9 +82,12 @@ namespace blueprint
         jassert(code.isNotEmpty());
         auto* ctxRawPtr = dukContext.get();
 
-        try {
+        try
+        {
```

Comment:
Either we're doing braces one way or the other... the mix of braces is gastly, especially in a public framework.

Reply:
Much prefer this style.